admin/api_access_tracking.go

```diff
@@ -11,7 +11,7 @@ import (
 	"strings"
 )
 
-type AccessTrackingApi interface {
+type AccessTrackingAPI interface {
```

Comment:
A pretty noticeable change. 
7.0  renames Api => API. It will break every possible SDK method call and example.

Reply:
Rename happened in java codebase making it hard to adjust change to become non breaking.